sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/sources/memoryV2.scala

```diff
@@ -178,7 +179,7 @@ class MemoryDataWriter(partition: Int, outputMode: OutputMode)
  * Used to query the data that has been written into a [[MemorySinkV2]].
  */
 case class MemoryPlanV2(sink: MemorySinkV2, override val output: Seq[Attribute]) extends LeafNode {
-  private val sizePerRow = output.map(_.dataType.defaultSize).sum
```

Comment:
@tdas @jose-torres Is that possible this can be zero?

Reply:
I wouldn't think it's possible.